.codex/mcp.json

```diff
@@ -0,0 +1,3 @@
+{
+  "servers": []
```

Comment:
This object key is atypical for MCP configurations. Refer to https://github.com/microsoft/playwright-mcp where we see the expected configuration of

```
{
  "mcpServers": {
    "playwright": {
      "command": "npx",
      "args": [
        "@playwright/mcp@latest"
      ]
    }
  }
}
```

What value does creating user confusion create by not calling this key "mcpServers"?